crates/pop-cli/src/commands/build/parachain.rs

```diff
@@ -1,34 +1,24 @@
 // SPDX-License-Identifier: GPL-3.0
 
 use crate::{cli, style::style};
-use clap::Args;
 use pop_common::Profile;
 use pop_parachains::build_parachain;
 use std::path::PathBuf;
 #[cfg(not(test))]
 use std::{thread::sleep, time::Duration};
 
-#[derive(Args)]
-pub struct BuildParachainCommand {
-	/// Directory path for your project [default: current directory].
-	#[arg(long)]
-	pub(crate) path: Option<PathBuf>,
+// Represents the configuration for building a parachain.
+pub struct BuildParachain {
+	/// Directory path for your project.
```

Comment:
```suggestion
	/// Path of the parachain project.
```

Reply:
This isn’t a change in this PR, so I prefer to leave the previous comment for now.

Reply:
It is a very small change and makes it consistent with the other command (which is changed)